### DISCUSSION TITLE
Using Images in an Embedded README with PackageReadmeFile

### DISCUSSION THREAD

**Discussion by RehanSaeed** [NuGet/Home#11307] [Q&A]
I want to embed a README.md file in my NuGet package using the new `PackageReadmeFile` setting. My README contains images. 

1. How do I know what images will get displayed? 
2. Can I embed an image using this syntax:
    ```xml
    <None Include="..\..\Images\Banner.png" Pack="true" PackagePath="\" />
    ```
    And then use the image in my README like so?
    ```md
    ![Banner](Images/Banner.png)
    ```
3. Is there a list of domains where external images can be loaded from?

**Comment by erdembayar** [comment]
@loic-sharma 
#3 Looks server team domain.

**Comment by erdembayar** [comment]
Currently readme.md doesn't support embedding images.
